lib/stelace.js

```diff
@@ -179,6 +181,27 @@ export class Stelace {
       this[key] = new resources[name](this)
     }
   }
+
+  onError (type, callback) {
+    if (!errorTypes.includes(type)) throw new Error(`Invalid error type: ${type}`)
+
+    const allListeners = this.getApiField('errorListeners')
+    allListeners[type] = allListeners[type] || []
+    allListeners[type].push(callback)
+
+    return function () {
+      allListeners[type] = allListeners[type].filter(cb => cb !== callback)
+    }
```

Comment:
You’re not documenting (in readme nor inline with one simple word `return function () { // unsubscribe`) how returned function should be used.

Reply:
Thanks, I'll add it in readme.

Reply:
After reading twice, I wonder why we don’t simply name this anonymous function `unsubscribe`, probably better for potential error stacks, and self-documenting.

Reply:
And you may want to add a test for this too, like having two listeners, unsubscribing one, and checking only the other is called after error.

Reply:
True. I added a test to check unsubscribe functions works correctly.